### PR TITLE
Add Coinut

### PR DESCRIPTION
**Pull request by wangxinxi**

### CODE REVIEW

**Comment by mxxcon** [comment]
Thank you for the submission, but at this point in time is we have a cut off limit of Alexa top 200k sites and this site is below that https://www.alexa.com/siteinfo/coinut.com

**Comment by kilasuit** [comment]
Again this is a Silly moot point 

Rethink this and change the decision here team

**Comment by mxxcon** [comment]
@kilasuit We have a limited number people managing this project who have limited amount of free time and attention. At this point in time we limit new PRs to have the maximum possible impact. We settled on criteria of alexa top200k sites.

**Comment by wangxinxi** [comment]
@mxxcon We are a crypto trading exchange with huge amount of asset belonging to our users. It's not good to compare us with other normal sites using the same metric. Also, we used to be top200k and are still near it.

**Comment by mxxcon** [comment]
@wangxinxi if that is not a good metric, then what is a good metric to use?

**Comment by wangxinxi** [comment]
Should also consider the industry categories. Crypto exchanges involve a lot of money and should be protected more.

**Comment by mxxcon** [comment]
Again, what kind of metric is good to use?
But please don't expect us to go digging into every minutia of every industry.
We are not going to be digging into stuff such as market capital of every exchange. There's no single readily available source to compare them all equally.

**Comment by wangxinxi** [comment]
We are now a top 200K site https://www.alexa.com/siteinfo/coinut.com. Please approve this.

File: _data/cryptocurrencies.yml [hunk 1]
```diff
@@ -186,6 +186,13 @@ websites:
       software: Yes
       doc: https://support.coins.ph/hc/en-us/articles/202604924-What-is-two-factor-authentication-
 
+    - name: COINUT
+      url: https://coinut.com/
+      img: coinut.png
+      tfa: Yes
+      software: Yes
+      doc: https://help.coinut.com/hc/en-us/articles/360005061354-What-is-Two-Factor-Authentication-2FA-
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]: Please change the doc url to `https://help.coinut.com/hc/en-us/articles/360005061354`

File: _data/cryptocurrencies.yml [hunk 2]
```diff
@@ -186,6 +186,13 @@ websites:
       software: Yes
       doc: https://support.coins.ph/hc/en-us/articles/202604924-What-is-two-factor-authentication-
 
+    - name: COINUT
```
Comment (Carlgo11) [inline _data/cryptocurrencies.yml]: On the main page title it says "Coinut" and not "COINUT". Which spelling is the right one?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/cryptocurrencies.yml (hunk 2 above)
Comment (wangxinxi) [reply _data/cryptocurrencies.yml]: Changed to Coinut in the new commit.

File: _data/cryptocurrencies.yml (hunk 1 above)
Comment (wangxinxi) [reply _data/cryptocurrencies.yml]: Chagned in the new commit.

**Review by Carlgo11 — DISMISSED** [review]

**Review by mxxcon — DISMISSED** [review]

**Review by Carlgo11 — DISMISSED** [review]

**Comment by Carlgo11** [comment]
So I can't seem to get Travis to run 😕 
travis-screenshot

**Review by Carlgo11 — APPROVED** [review]
Alright! The Travis team have fixed the problem. Apparently the system had flagged the repo/pr as spam 🤔

**Review by jdavis — APPROVED** [review]

**Pull request merged**